### DISCUSSION TITLE
LoadResources module: Invalid Face Specification

### DISCUSSION THREAD

**Discussion by chiyu1203** [bonsai-rx/bonsai#1981] [Q&A] [answered]
Dear Bonsai Discussion,
I am trying to build up a VR workflow where virutal locusts are rendered in the environment. My workflow can load simple 3D object. However, when it comes to locust models, LoadResource module returned Runtime error, saying Invalid Face Specification: All faces must have the same number of vertices. It seems that the problem came from the locust 3D models. However, I can load these models on Unity. Therefore, I am curious if there is something I dont know about MeshResources and LoadResources. Please feel free to let me know if you have any thought.
Below is the website where I download the 3D models. 
https://free3d.com/3d-model/locust-v1--32618.html
https://free3d.com/3d-model/locust-7471.html
Best Regards
Chi-Yu

**Comment by Jorkdekok1** [comment, marked as answer]
Hey Chi-Yu, 

The error message that you get is I think because you didn’t put the triangulate mesh when exporting the file from blender to an Obj file. Also you will need to use the bindtexture node right before the drawscene model node. The draw scene model node does seem to allow you to bind  textures. Within the bind texture node you need to specify the texture you want to bind to the model and the shadername of the model (which is textured model).  I did have problems sometimes when I did try and bind textures like this. 
 
![image](https://github.com/user-attachments/assets/e19211f0-9688-43f3-aee8-b0dc79fdbeff)

![image](https://github.com/user-attachments/assets/b3781c74-310b-4d6a-a632-05484c23ffad)

Kind regards, 
Jork

**Reply by chiyu1203** [reply]
You are right! That solved my problem! Thank you so much!
